src/systemctl/systemctl-describe.c

```diff
@@ -0,0 +1,35 @@
+/* SPDX-License-Identifier: LGPL-2.1-or-later */
+
+#include "bus-error.h"
+#include "bus-locator.h"
+#include "systemctl.h"
+#include "systemctl-util.h"
+#include "systemctl-describe.h"
+
+int verb_describe(int argc, char *argv[], void *userdata) {
```

Comment:
hmm, I am not sure this should be a first class verb. Instead, maybe expose this as "systemctl show --json=pretty", i.e. as the json mode of "systemctl show"